src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();

```

Comment:
Static methods are in general a bad practice. Let's better create an instance of a class which method you want to call. You should create an instance of the `Random` class at the class level, not inside the method.